text/0000-Project-Goals-2025h1.md

```diff
@@ -0,0 +1,662 @@
+| [Finish the libtest json output experiment](https://rust-lang.github.io/rust-project-goals/2025h1/libtest-json.html)                                                | [Ed Page][]           | [testing-devex]                                     |
+| [Implement Open API Namespace Support](https://rust-lang.github.io/rust-project-goals/2025h1/open-namespaces.html)                                                  | ![Help Wanted][] | [cargo], [compiler]                                 |
+| [Implement restrictions, prepare for stabilization](https://rust-lang.github.io/rust-project-goals/2025h1/restrictions.html)                                        | [Jacob Pratt][]         | [compiler], [lang]                                  |
+| [Improve state machine codegen](https://rust-lang.github.io/rust-project-goals/2025h1/improve-rustc-codegen.html)                                                   | [Folkert de Vries][]      | [compiler], [lang]                                  |
+| [Instrument the Rust standard library with safety contracts](https://rust-lang.github.io/rust-project-goals/2025h1/std-contracts.html)                              | [Celina G. Val][]        | [compiler], [libs]                                  |
+| [Integration of the FLS into the Rust Project](https://rust-lang.github.io/rust-project-goals/2025h1/spec-fls-integration.html)                                     | [Joel Marcey][]      | [release], [spec]                                   |
+| [Making compiletest more maintainable: reworking directive handling](https://rust-lang.github.io/rust-project-goals/2025h1/compiletest-directive-rework.html)       | [Jieyou Xu][]        | [bootstrap], [compiler], [rustdoc]                  |
+| [Metrics Initiative](https://rust-lang.github.io/rust-project-goals/2025h1/metrics-initiative.html)                                                                 | [Jane Lusby][]           | [compiler], [infra]                                 |
+| [Model coherence in a-mir-formality](https://rust-lang.github.io/rust-project-goals/2025h1/formality.html)                                                          | [Niko Matsakis][]    | [types]                                             |
+| [Next-generation trait solver](https://rust-lang.github.io/rust-project-goals/2025h1/next-solver.html)                                                              | [lcnr][]            | [types]                                             |
+| [Nightly support for ergonomic SIMD multiversioning](https://rust-lang.github.io/rust-project-goals/2025h1/simd-multiversioning.html)                               | [Luca Versari][]        | [lang]                                              |
+| [Null and enum-discriminant runtime checks in debug builds](https://rust-lang.github.io/rust-project-goals/2025h1/null-enum-discriminant-debug-checks.html)         | [Bastian Kersting][]          | [compiler], [lang], [opsem]                         |
```

Comment:
Where is the right place to ask clarifying questions about a goal? Seems a bit messy to do it all in this PR, but since I can't find a better place...

It is a bit unclear to me what is asked from us here in terms of a "design meeting". Like, what is there to design, opsem-wise? The opsem rules are fairly clear for this.

Reply:
Yeah I figured the design meeting is not needed already. I can just remove that if wanted.

Reply:
I also wondered whether that meeting would be needed.